kitsune/flagit/views.py

```diff
@@ -163,9 +171,26 @@ def build_hierarchy(parent_id=None, level=0):
 
 
 @group_required("Content Moderators")
+@require_http_methods(["GET", "POST"])
 def moderate_content(request):
     """Display flagged content that needs moderation."""
     product_slug = request.GET.get("product")
+    assignee = request.GET.get("assignee")
+
+    if request.method == "POST":
+        if not (assignee and (request.user.username == assignee)):
+            return HttpResponseForbidden()
+        action = request.POST.get("action")
+        if not (action and (action in ("assign", "unassign"))):
+            return HttpResponseBadRequest()
+    else:
+        if (
```

Comment:
You can move this above the check if the request is `POST` to avoid the nesting. This will also help you to have a single check of 
```python
if request.method == "POST":
```